notebooks/exploring_explorer.livemd

```diff
@@ -964,7 +964,7 @@ series contents, but every operation will be optimized and run only once.
 Remember those helpful error messages?
 
 ```elixir
-DF.filter(df, cuontry == "BRAZIL")
+DF.filter(df, country == "BRAZIL")
```

Comment:
This typo is intentional, because we want to show the error message.

```suggestion
DF.filter(df, cuontry == "BRAZIL")
```

Reply:
I've removed this change. Thanks!